src/views/docs/en/guides/developer-experience/continuous-integration/github-actions.md

```diff
@@ -4,28 +4,65 @@ category: Continuous integration
 description: Deploy an Architect project from GitHub Actions.
 ---
 
-Deploy an Architect project from GitHub Actions. This particular example (inspired by [the Remix team's implementation](https://github.com/remix-run/grunge-stack/blob/bc9270eb29eda1a806d6b4c773ebcf84f216e2ab/.github/workflows/deploy.yml)) will deploy commits to the `dev` branch to staging and commits to `main` to production. Extract or add steps as needed for your pipeline.
+Architect projects can be tested and deployed from GitHub Actions.
 
-Set up this action by adding to your project's repository in `./.github/workflows/deploy.yml`.
+## Architect-provided actions
 
-> 🔑  Required: Set your project's `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in [your GitHub repository's secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets).  
-<!-- Alternatively, your GitHub Action can [gather keys from a specific AWS IAM Role federated by an IAM OIDCProvider](./github-actions-iam-oidcp). -->
+Architect has created [`architect/action-build`](https://github.com/architect/action-build) and [`architect/action-deploy`](https://github.com/architect/action-deploy) for GitHub Actions. These can be included as a part of your project's workflows.
 
-## Action YAML template
+> 🔑  Required: Set your project's `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in [your GitHub repository's secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets). 
```

Comment:
Minor quibble, it can be the GitHub organization secrets or repositories secrets.